src/ol/style/text.js

```diff
@@ -96,6 +96,28 @@ ol.style.Text.DEFAULT_FILL_COLOR_ = '#333';
 
 
 /**
+ * Clones the style.
+ * @return {ol.style.Text} The cloned style.
+ * @api
+ */
+ol.style.Text.prototype.clone = function() {
+  return new ol.style.Text({
+    font: this.getFont(),
+    rotation: this.getRotation(),
+    rotateWithView: this.getRotateWithView(),
```

Comment:
That was added recently, are you sure you checked the most up to date documentation?